Server/Ebenezer/User.cpp

```diff
@@ -11180,7 +11275,7 @@ void CUser::ClientEvent(char* pBuf)
 			break;
 
 		case NPC_CAPTAIN:
-			eventid = 15002;
+			eventid = (m_pUserData->m_bNation == KARUS ? 15001 : 15002);
```

Comment:
This is not official behaviour; the previous implementation was already updated to match official behaviour here.
This means that the accompanying EVT changes will only behave with this server, and not the official server.
The EVT should be reworked for official behaviour, where its entry point has to be event 15002 -- you can add nation checks under that -- in the script -- to show appropriate menus.